.gitignore

```diff
@@ -1,5 +1,4 @@
 /target
-Cargo.lock
```

Comment:
This is a library crate. Why are we no longer ignoring `Cargo.lock`?

Reply:
the guidance has changed on this recently https://blog.rust-lang.org/2023/08/29/committing-lockfiles.html but if you prefer to keep it out I can fix it up

Reply:
Alright, I'll think about this.